### ISSUE TITLE
File method `copyTo` always returns `true`

### ISSUE THREAD

**Issue by DyaGa** [contao/core-bundle#1401]
As described in method description

```php
	/**
	 * Copy the file
         * ... 
	 * @return boolean True if the operation was successful
	 */
```

But this method always returns `true` even if file doesn't copy. See [File::copyTo](https://github.com/contao/core-bundle/blob/master/src/Resources/contao/library/Contao/File.php#L708-L742)

**Comment by leofeyer** [comment]
That's probably because the method always copies. What is your exact scenario?

**Comment by DyaGa** [comment]
The scenario is simple:

```php
if (!$file->copyTo(...)) {
  // throw exception
}
```

I mean, that expected result of this method should be `true` or `false`, but not always `true`, the same result as for php [copy function](https://secure.php.net/manual/en/function.copy.php)

**Comment by leofeyer** [comment]
Fixed in 7267ae977baac9dac841fec2eae3a1f31fca86b9.
